### PR TITLE
Changes to resolve issue with clusterpool object patch

### PR DESCRIPTION
**Pull request by yashoza19**

### CODE REVIEW

**Comment by mrhillsman** [comment]
[]byte(`[{"op": "replace", "path": "/spec/runningCount", "value": 0 },{"op": "replace", "path": "/spec/size", "value": 0}]`)

i could have done it the way you suggested but it is more code than necessary. patch is just an http request essentially and the type changes the content-type header for the request. the body can be represented in many ways which i thought i formatted properly. here is the documentation on the type i set: https://datatracker.ietf.org/doc/html/rfc6902; more specifically https://datatracker.ietf.org/doc/html/rfc6902#section-3 which is the structure of the "document"/body of the request

possible types are linked here - https://kubernetes.io/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch/#use-a-json-merge-patch-to-update-a-deployment

basically i was using the merge type document format but set the type of the document i was sending to json
